website/docs/distributed-databases/cloudnative-postgres.md

```diff
@@ -1,9 +1,16 @@
 ---
 sidebar_position: 2
+<<<<<<< HEAD
+sidebar_label: CloudNativePG PostgreSQL
+---
+
+# PostgresSQL database on EKS with CloudNativePG Operator
```

Comment:
Change this to `Deploying PostgreSQL Database on EKS using CloudNativePG Operator`